runcellpose.py

```diff
@@ -72,13 +74,27 @@ def create_settings(self):
 
         self.mode = Choice(
             text="Detection mode",
-            choices=[MODE_NUCLEI, MODE_CELLS, MODE_CUSTOM],
-            value=MODE_NUCLEI,
+            choices= model_dic,
+            value=model_dic[2],

```

Comment:
Can we keep the default value as a string rather than a part of the list? The former will only break if they delete that model entirely, the latter might sneaky-change our default if the Cellpose folks reorder the list. If it was a list WE had control over, I wouldn't care, but since we don't, this is a bit risky.